### ISSUE TITLE
LoadError: type StackFrame has no field fromC

### ISSUE THREAD

**Issue by maximerischard** [timholy/ProfileView.jl#50]
``` julia
using ProfileView

function profile_test(n)
    for i = 1:n
        A = randn(100,100,20)
        m = maximum(A)
        Afft = fft(A)
        Am = mapslices(sum, A, 2)
        B = A[:,:,5]
        Bsort = mapslices(sort, B, 1)
        b = rand(100)
        C = B.*b
    end
end

profile_test(1)  # run once to trigger compilation
Profile.clear()  # in case we have any previous profiling data
@profile profile_test(10)

ProfileView.view()
```

yields

```
LoadError: type StackFrame has no field fromC
```

**Comment by maximerischard** [comment]
This is with a very recent version of julia 0.5.0. It might not be worth chasing changes in the julia API.

```{julia}
versioninfo()
```

```
Julia Version 0.5.0-dev+2496
Commit f020d38* (2016-02-03 20:27 UTC)
Platform Info:
  System: Darwin (x86_64-apple-darwin14.5.0)
  CPU: Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libopenlibm
  LLVM: libLLVM-3.7.1
```

```{julia}
Pkg.status("ProfileView")
```

```- ProfileView                   0.1.1```

**Comment by timholy** [comment]
I may get to this eventually, but 0.5 is currently intended for "users who can fix bugs in the packages they depend on." :smile:

**Comment by Zac12345** [comment]
Changing line 90 of Profileview from : 
`isjl = builddict(uip, [!lkup[i].fromC for i = 1:nuip])`

to 

`isjl = builddict(uip, [!lkup[i].from_c for i = 1:nuip])`

will fix this for you

**Comment by timholy** [comment]
Fixed by #51.
